pkg/stack/stack.go

```diff
@@ -195,6 +195,12 @@ var DHIS2Core = model.Stack{
 		"STARTUP_PROBE_PERIOD_SECONDS":    {Priority: 12, DisplayName: "Startup Probe Period Seconds", DefaultValue: &dhis2CoreDefaults.startupProbePeriodSeconds},
 		"JAVA_OPTS":                       {Priority: 13, DisplayName: "JAVA_OPTS", DefaultValue: &dhis2CoreDefaults.javaOpts},
 		"CHART_VERSION":                   {Priority: 14, DisplayName: "Chart Version", DefaultValue: &dhis2CoreDefaults.chartVersion},
+		"GOOGLE_AUTH_PROJECT_ID":          {Priority: 15, DisplayName: "Chart Version", DefaultValue: &dhis2CoreDefaults.chartVersion},
```

Comment:
Do the google parameters need a default? And if so why is it 
`&dhis2CoreDefaults.chartVersion`?